webapp/channels/src/components/work_templates/components/channel_only_footer.tsx

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {useDispatch} from 'react-redux';
+import {useIntl} from 'react-intl';
+import styled from 'styled-components';
+import classnames from 'classnames';
+
+import {ModalIdentifiers, TELEMETRY_CATEGORIES} from 'utils/constants';
+import {closeModal} from 'actions/views/modals';
+import {trackEvent} from 'actions/telemetry_actions';
+import {DispatchFunc} from 'mattermost-redux/types/actions';
+
+interface Props {
+    createChannel: () => void;
+    isConfirmDisabled: boolean;
+}
+
+export default function ChannelOnlyFooter(props: Props) {
+    const {formatMessage} = useIntl();
+    const dispatch = useDispatch<DispatchFunc>();
+    const trackAction = (action: string, actionFn: () => void) => () => {
+        trackEvent(TELEMETRY_CATEGORIES.WORK_TEMPLATES, action, props);
+        actionFn();
+    };
+    const createChannelOnly = trackAction('btn_go_to_customize', props.createChannel);
+    const cancelButtonAction = trackAction('close_channel_only', () => {
+        dispatch(closeModal(ModalIdentifiers.WORK_TEMPLATE));
+    });
+
+    return (
+        <Footer>
+            <FooterButton
+                cancel={true}
+                onClick={cancelButtonAction}
+            >
+                {formatMessage({id: 'work_templates.channel_only.cancel', defaultMessage: 'Cancel'})}
+            </FooterButton>
+            <FooterButton
+                confirm={true}
+                onClick={createChannelOnly}
+                disabled={props.isConfirmDisabled}
+                className={classnames({disabled: props.isConfirmDisabled})}
+            >
+                {formatMessage({id: 'work_templates.channel_only.confirm', defaultMessage: 'Create channel'})}
+            </FooterButton>
+        </Footer>
+    );
+}
+
+const genericModalSidePadding = '32px';
+
+const Footer = styled.div`
+    position: relative;
+    &:after {
+        content: '';
+        position: absolute;
+        width: calc(100% + ${genericModalSidePadding} * 2);
+        left: -${genericModalSidePadding};
+        top: 0;
+        height: 1px;
+        background-color: rgba(var(--center-channel-text-rgb), 0.08);
+    }
+    padding: 24px 0;
+    text-align: right;
+`;
+
+interface FooterButtonProps {
+    cancel?: boolean;
+    confirm?: boolean;
+}
+const primaryButton = `
```

Comment:
Added a scss file and used the mixins that were available. It still copies some styles from the generic modal for the footer buttons, but I don't want to spend time extracting that out to something more reusable since that modal is being deprecated in favor of the MUI-based modal. [Briefly tried](https://github.com/mattermost/mattermost-server/pull/23189/commits/d67bf7f6689c00105278db9d3a464489c9b78998) using the up and coming MUI-based buttons which would have been nice, but they are [not quite ready](https://github.com/mattermost/mattermost-server/pull/23308) yet.